### ISSUE TITLE
CSS suffix attribute selector is showing as an error

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#639]
(Original issue 932 created by mottie on 2013-11-25T23:03:01.809190+00:00)

Of all the [css attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors), the one that shows an error is the suffix selector ([example](http://pygments.org/demo/149902/)):

```
#!css

/* All links to urls ending in ".cn" are red */
a[href$=".cn"] {color: red;}
```

**Comment by Anteru** [comment]
(Original issue 932 last updated on 2014-01-09T17:04:48.545742+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T17:02:43.668336+00:00)

fixed by b81d0f189932

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T17:04:48.538673+00:00)

css: add "$" to possible operators (fixes #932)

→ <<cset b81d0f189932>>
